examples/verif-manual/vm-295.py

```diff
@@ -0,0 +1,268 @@
+# Specify load set to read from the result file, load step =1, sub-step=46
+mapdl.set(1, 46)
+p31 = mapdl.get("P31", "NODE", nd1, "PRES")
+t31 = mapdl.get("T31", "ACTIVE", 0, "SET", "TIME")
+# Specify load set to read from the result file, load step =1, sub-step=47
+mapdl.set(1, 47)
+p32 = mapdl.get("P32", "NODE", nd1, "PRES")
+t32 = mapdl.get("T32", "ACTIVE", 0, "SET", "TIME")
+t3 = day * 0.3
+mapdl.com("")
+mapdl.com("INTERPOLATE THE RESULTS AT LOCATION (1,6,0) FOR TIME=0.3DAY")
+mapdl.com("")
+pt3 = (p31 + (t3 - t31) / (t32 - t31) * (p32 - p31)) / 1e3
+mapdl.vfill("P(3)", "DATA", pt3)
+# Specify load set to read from the result file, load step =1, sub-step=61
+mapdl.set(1, 61)
+p41 = mapdl.get("P41", "NODE", nd1, "PRES")
+t41 = mapdl.get("T41", "ACTIVE", 0, "SET", "TIME")
+# Specify load set to read from the result file, load step =1, sub-step=62
+mapdl.set(1, 62)
+p42 = mapdl.get("P42", "NODE", nd1, "PRES")
+t42 = mapdl.get("T42", "ACTIVE", 0, "SET", "TIME")
+t4 = day * 0.4
+mapdl.com("")
+mapdl.com("INTERPOLATE THE RESULTS AT LOCATION (1,6,0) FOR TIME=0.4DAY")
+mapdl.com("")
+pt4 = (p41 + (t4 - t41) / (t42 - t41) * (p42 - p41)) / 1e3
+mapdl.vfill("P(4)", "DATA", pt4)
+# Specify load set to read from the result file, load step =1, sub-step=76
+mapdl.set(1, 76)
+p51 = mapdl.get("P51", "NODE", nd1, "PRES")
+t51 = mapdl.get("T51", "ACTIVE", 0, "SET", "TIME")
+# Specify load set to read from the result file, load step =1, sub-step=77
+mapdl.set(1, 77)
+p52 = mapdl.get("P52", "NODE", nd1, "PRES")
+t52 = mapdl.get("T52", "ACTIVE", 0, "SET", "TIME")
+t5 = day * 0.5
+mapdl.com("")
+mapdl.com("INTERPOLATE THE RESULTS AT LOCATION (1,6,0) FOR TIME=0.5DAY")
+mapdl.com("")
+pt5 = (p51 + (t5 - t51) / (t52 - t51) * (p52 - p51)) / 1e3
+mapdl.vfill("P(5)", "DATA", pt5)
+
+# Enter non-interactive mode
+with mapdl.non_interactive:
+    # REFERENCE RESULTS, FIGURE 5, PG 5916
+    mapdl.dim("CP", "", 5)
+    mapdl.run("CP(1)=5.230,2.970,1.769,1.043,0.632  ")
+    mapdl.dim("RT", "", 5)
+    mapdl.run("*DO,I,1,5")
+    mapdl.run("RT(I)=P(I)/CP(I)")
+    mapdl.run("*ENDDO")
+    mapdl.dim("LABEL", "CHAR", 5)
+    mapdl.run("LABEL(1)='0.1','0.2','0.3','0.4','0.5'")
+    mapdl.com("")
+    mapdl.run("/OUT,vm295,vrt")
+    mapdl.com("------------ vm295 RESULTS COMPARISON --------------")
+    mapdl.com("")
+    mapdl.com("Time     |  TARGET   |  Mechanical APDL  |  RATIO")
+    mapdl.com("")
+    mapdl.run("*VWRITE,LABEL(1),CP(1),P(1),RT(1)")
+    mapdl.run("(8X,A4,'       ',F10.3,'       ',F10.3,'    ',F10.2)")
+    mapdl.com("")
+    mapdl.com("------------------------------------------------------")
+    # redirects output to the default system output file
+    mapdl.run("/OUT")
+    # reactivates suppressed printout
+    mapdl.gopr()
+
+# Get the mapdl temporary working directory
+vrt_file_path = os.path.join(mapdl.directory, "vm295.vrt")
+
+# read the vm295.vrt file to print the results
+f = open(vrt_file_path, "r")
+for x in f:
+    print(x)
```

Comment:
You can use the same method as for `vm13.py`

Reply:
Done